spec/webmock/api_get_repo_pages.json

```diff
@@ -1 +1 @@
-{"message":"Not Found","documentation_url":"https://developer.github.com/v3"}
\ No newline at end of file
+{"message":"Not Found","documentation_url":"https://developer.github.com/v3"}
```

Comment:
I didn't modify this. Restored the whitespace via 43e425b to clean up the diff. 

Based on trying to implement things differently the first time, I believe this is here to ensure that repo compat is exercised for the integration test.